app/main/updater.js

```diff
@@ -0,0 +1,56 @@
+import {dialog} from 'electron';
+import {autoUpdater} from 'electron-updater';
+
+import {t} from './helpers';
+
+const RELEASES_LINK = 'https://github.com/appium/appium-inspector/releases';
+
+autoUpdater.autoDownload = false;
+autoUpdater.autoInstallOnAppQuit = false;
+
+autoUpdater.on('error', (error) => {
+  dialog.showErrorBox(t('Could not download update'), t('updateDownloadFailed', {message: error}));
+});
+
+autoUpdater.on('update-not-available', () => {
+  dialog.showMessageBox({
+    type: 'info',
+    buttons: [t('OK')],
+    message: t('No update available'),
+    detail: t('Appium Inspector is up-to-date'),
+  });
+});
+
+autoUpdater.on('update-available', async ({version, releaseDate}) => {
+  const pubDate = new Date(releaseDate).toDateString();
+  const {response} = await dialog.showMessageBox({
+    type: 'info',
+    message: t('appiumIsAvailable', {name: version}),
+    buttons: [t('Install Now'), t('Install Later')],
+    detail: t('updateDetails', {pubDate, notes: RELEASES_LINK}),
+  });
+  if (response === 0) {
+    dialog.showMessageBox({
```

Comment:
not sure if await is missing

Reply:
if that is an async call then the next call will happen anyway unless it is put into `then` callback

Reply:
I tested this just now with the await added - the download does not actually start until the dialog is dismissed. I believe the await causes Electron to wait for user input (in this case, dismissal of the dialog box)

Reply:
yep, that is why I wanted to know if you want to start the update download progress only after the dialog is accepted or asynchronously while it's shown

Reply:
I think it's better to start it asynchronously - from a user perspective, the message box text implies that the download has already started.

Reply:
then this code is OK. maybe just add a comment that this behaviour is expected